pyproject.toml

```diff
@@ -33,10 +33,10 @@ dependencies = [
     "crds >=13.0.2",
     "drizzle >= 2.1.1,<2.2",
     "gwcs >=0.25.2,<0.27.0",
-    "stcal>=1.15.2,<1.16.0",
+    "stcal @ git+https://github.com/spacetelescope/stcal.git",
+    #"stcal>=1.15.2,<1.16.0",
     "stpipe >=0.11.0,<0.12.0",
     "spherical-geometry >=1.3.3,<1.4",
-    "stsci.imagestats >=1.8.3,<1.9",  # remove when stcal fixes this missing dep
```

Comment:
I removed this since this PR switch stcal to main, which has imagestats as a dependency:
https://github.com/spacetelescope/stcal/blob/8cbe0a3c69135022ff30611d21ac5af1bda4258a/pyproject.toml#L23